Cargo.toml

```diff
@@ -9,10 +9,15 @@ readme = "README.md"
 license = "Apache-2.0/MIT"
 categories = ["data-structures"]
 
+[features]
+bincode = ["dep:bincode", "vob/bincode", "packedvec/bincode"]
+serde = ["dep:serde", "vob/serde", "packedvec/serde"]
+
 [dependencies]
-vob = { version=">=3.0.2", features=["serde"] }
-packedvec = { version="1.0", features=["serde"] }
+vob = { version=">=3.0.4" }
```

Comment:
Do we need the `>=`? If so, we need it on `packedvec` too I suppose? Or we need it on neither, and just specifying the version number is equivalent to `>=`?

Reply:
I was just mirroring how it was, and was hoping we could remove it.